sklearn/ensemble/tests/test_forest.py

```diff
@@ -234,8 +271,8 @@ def test_poisson_vs_mse():
         # If y = 0 for those, the poisson deviance gets too good.
         # If we drew more samples, we would eventually get y > 0 and the
         # poisson deviance would explode, i.e. be undefined. Therefore, we do
-        # not clip to a tiny value like 1e-15, but to 1e-6. This acts like a
-        # small penalty to the non-positive predictions.
+        # not clip to a tiny value like 1e-15, but to 0.1. This acts like a
+        # mild penalty to the non-positive predictions.
```

Comment:
It seems the merge was not clean.
This is unrelated to this PR and was changed by [another commit](https://github.com/scikit-learn/scikit-learn/commit/2b15b908c11b90a15253394b1a03bd535720d6ce) already in main

Reply:
fixed